lib/handler.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const _ = require('lodash');
+const aeclient = require('aws-elasticsearch-client');
+const errors = require('./errors');
```

Comment:
I always did like that at CT, but here I've never seen anyone to follow this practice.. But yes, I prefer to separate requires into to groups.